airflow/www/static/footable.core.css

```diff
@@ -0,0 +1,218 @@
+@font-face {
```

Comment:
It's a 3rd party library, is it ok to add apache license? What if the file already has a license of different type?